extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -74,6 +74,16 @@ class ActivityHistory {
      * @private {ActivityEventDef|undefined}
      */
     this.prevActivityEvent_ = undefined;
+
+    /**
+     * Contains the incrementalEngagedTime timestamps for named timers.
+     * @private {Object}
+     */
+    this.incrementalEngagedTime_ = {
```

Comment:
I feel like the naming is a little bit confusing. The object stores the last totalEngagedTime_ under same name from last request, but not incrementalTime. Could we rename it?